src/components/Main/RestaurantList.jsx

```diff
@@ -12,6 +12,7 @@ function RestaurantList( {restaurants} ) {
                         description={restaurant.description}
                         category={restaurant.category}
                         alt={restaurant.alt}
+                        onClick={() => onChangeModal(restaurant.name)}
```

Comment:
```suggestion
                        onClick={() => onChangeModal(restaurant.name, restaurant.description)}
```
이런 식으로 설명도 같이 전달하면 App에서 레스토랑을 찾지 않아도 될 거 같은데 name만 전달하신 이유가 있으실 거 같은데 궁금하네요..!

Reply:
오, 제가 놓친 부분 같아요!! 감사합니다